doc/source/cluster/kubernetes/user-guides/k8s-cluster-setup.md

```diff
@@ -9,9 +9,9 @@ aws-eks-gpu-cluster
 gcp-gke-gpu-cluster
 ```
 
-The KubeRay operator and Ray can run on any cloud or on-prem Kubernetes cluster.
-The simplest way to provision a remote Kubernetes cluster is to use a cloud-based managed service.
-We collect a few helpful links for users who are getting started with a managed Kubernetes service.
+Most KubeRay documentation only requires a local Kubernetes cluster such as [Kind](https://kind.sigs.k8s.io/).
```

Comment:
I think kind officially call themselves just `kind` (all lower caps)

Reply:
I knew it. All KubeRay documentation uses `Kind` instead of `kind` to indicate that it is a proper noun rather than a typo. I found that some KubeRay users haven’t heard about Kind before.

Reply:
Thank you for the review!